services/httpd/handler_test.go

```diff
@@ -220,7 +220,7 @@ func TestHandler_Query_Auth(t *testing.T) {
 	h.ServeHTTP(w, req)
 	if w.Code != http.StatusUnauthorized {
 		t.Fatalf("unexpected status: %d: %s", w.Code, w.Body.String())
-	} else if !strings.Contains(w.Body.String(), `{"error":"Token is expired`) {
+	} else if !strings.Contains(w.Body.String(), `{"error":"token is expired`) {
```

Comment:
Probably shouldn't change the error message, if you don't know if something like Chronograf is relying on it

Reply:
i'm not changing the error message -- the go-jwt package changed *it's* error message.  this change is in a test to reflect this change.  the same error is occurring but the verbiage of the error output has changed.